### ISSUE TITLE
Public method which should have been internal

### ISSUE THREAD

**Issue by arberg** [Nets-mobile-acceptance/Netaxept-Android-SDK#55]
SDK: 2.6.4

I noticed this public method, as it pops up as auto-complete choice in AndroidStudio

```
// IntelliJ API Decompiler stub source generated from a class file
// Implementation of methods is not available

package eu.nets.pia

public fun kotlin.Any.assertDegub(expected: kotlin.Boolean, message: kotlin.String): kotlin.Unit { /* compiled code */ }

```

Could you make that an internal method? Probably also best to check if you have other similar stuff that shouldn't have been public.

**Comment by Miss-PiA** [comment]
@arberg This issue will be solved in the next major release. We will keep you posted once the release is available.

**Comment by Miss-PiA** [comment]
Hi @arberg ,
Issue has been fixed in Netaxept-Android-SDK v2.7.0.
Thanks
